README.md

```diff
@@ -228,8 +228,11 @@ for production on Linux, but it can be useful for development or easier integrat
 The following commands run the agent outside of Docker:
 
 ```
-make gobuild
-./out/amazon-ecs-agent
+sudo yum install rpm-build
+make generic-rpm
+sudo yum localinstall amazon-ecs-agent-1.53-1.x86_64.rpm
+sudo vi /etc/ecs/ecs.config #add environment variables (ECS_CLUSTER and ECS_DATADIR required)
```

Comment:
I think ecs-init used to define a default for ECS_DATADIR, should we define that default in agent now?

Reply:
The current solution for the environment variables requires pretty much all of the variables to be set manually. This includes setting the new data directory  as well.
Setting up the default directory would come in a future commit.

Reply:
could you add a TODO for this?